lib/shopify_cli.rb

```diff
@@ -80,14 +78,19 @@ module ShopifyCli
   # The rest of this file outlines classes and modules required by the shopify-app-cli
   # application and CLI kit framework.
   # To understand how this works, read https://github.com/Shopify/cli-kit/blob/master/lib/cli/kit.rb
+
+  # ShopifyCli::Config
   autocall(:Config)   { CLI::Kit::Config.new(tool_name: TOOL_NAME) }
+  # ShopifyCli::Logger
   autocall(:Logger)   { CLI::Kit::Logger.new(debug_log_file: DEBUG_LOG_FILE) }
+  # ShopifyCli::Resolver
   autocall(:Resolver) do
     ShopifyCli::Core::HelpResolver.new(
       tool_name: TOOL_NAME,
       command_registry: ShopifyCli::Commands::Registry
     )
   end
+  # ShopifyCli::ErrorHandler
   autocall(:ErrorHandler) do
```

Comment:
Adding these comments just to make it easier for devs to find these objects, since they are just references to other objects. It's hard when first learning this codebase to know where they are defined and what they do.